frontend/app/assets/javascripts/top_containers.bulk.js

```diff
@@ -539,6 +539,36 @@ function BulkActionMerge(bulkContainerSearch) {
     AS.openCustomModal("bulkActionModal", "Merge Top Containers", AS.renderTemplate("bulk_action_merge", {
       selection: self.bulkContainerSearch.get_selection()
     }), 'full');
+
+    // Access modal1 DOM
+    const $mergeBtn = $("[data-js='merge']");
+    
+    $mergeBtn.on("click", function() {
+      // Set up data for form submission
+      const victims = self.bulkContainerSearch
+                          .get_selection()
+                          .map(container => ({
+                            uri: container.uri,
+                            display_string: container.display_string
+                          }));
+
+      const target = {};
+      
+      target.el = document.querySelector('input[name="target[]"]:checked');
+      target.display_string = target.el.getAttribute('aria-label');
+      target.uri = target.el.getAttribute('value');
+
+      // Remove modal1
+      $("#bulkActionModal").remove();
```

Comment:
As I understood the requirements for this, the prior merge modal should not disappear when the confirmation modal is called.  It should remain and be overlaid with the confirm modal in line with other places in the application somewhat like this:

![image](https://user-images.githubusercontent.com/15144646/72103142-9583ae00-32f6-11ea-9018-b6dc3ccfb874.png)

Reply:
The animated gif at the top has been updated w/ the smaller confirm modal.

The functionality still doesn't work without removing the first modal due to the bootstrap 3 conflict mentioned below.

Reply:
@lorawoodford What's that "Rapid Data ..." selection modal in the background (so as to replicate the view you screenshotted)?

Reply:
Never mind (although I found the Rapid Data Entry view).

Screenshot above updated after a couple more commits - got the multi-modal view working.

Reply:
This is resolved via d86e001